vnext/Shared/OInstance.cpp

```diff
@@ -509,19 +509,23 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       std::string bundlePath = (fs::path(m_devSettings->bundleRootPath) / jsBundleRelativePath).string();
       auto bundleString = FileMappingBigString::fromPath(bundlePath);
 #else
-      std::string bundlePath = (fs::path(m_devSettings->bundleRootPath) / (jsBundleRelativePath + ".bundle")).string();
```

Comment:
>(fs::path(m_devSettings->bundleRootPath) / (jsBundleRelativePath + ".bundle")).string(); [](http://example.com/codeflow?start=31&length=88)

It would be preferable to keep using the fs::path for file paths. The bundleRootPath may have inconsistent path separator which this class may help to fix.